parts/inc/ppphtest

```diff
@@ -680,7 +680,7 @@ my %p;
 my $fail = 0;
 for (@o) {
   my($name, $flags) = /^(\w+)(?:\s+\[(\w+(?:,\s+\w+)*)\])?$/ or $fail++;
-  exists $p{$name} and $fail++;
+  { no warnings 'uninitialized'; exists $p{$name} and $fail++; }
```

Comment:
This module is the only one that has to work in really early perls.  That means, before you could say 'no warnings'.  The paradigm elsewhere in the code appears to be
BEGIN { 'warnings'->unimport('uninitialized') if "$]" > '5.006' }

Reply:
I think `no  if "$]" > '5.006, warnings 'uninitialized';` would work